registry-scanner/go.mod

```diff
@@ -3,20 +3,81 @@ module github.com/argoproj-labs/argocd-image-updater/registry-scanner
 go 1.22.3
 
 require (
+	github.com/argoproj-labs/argocd-image-updater v0.15.1
```

Comment:
Will this sub-module build and work without dependency on image-updater? I think we should avoid depending on image-updater for easy consumption.

Reply:
I think I missed updating one import to the correct package leading to the addition of image-updater dependency. Updated the import to correct package and removed dependency on image-updater.